### ISSUE TITLE
Unknown Keras loss function logcosh

### ISSUE THREAD

**Issue by cbiroan** [deeplearning4j/deeplearning4j#8291]
#### Issue Description

Please describe our issue:
- Expected behavior: logcosh loss function is supported
- Actual behavior: Error message: org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unknown Keras loss function logcosh. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version - 1.0.0-beta5
* Platform information (OS, etc) - Windows 10
* CUDA version, if used - N/A
* NVIDIA driver version, if in use N/A

#### Additional Information

Where applicable, please also provide:

* Full log or exception stack trace (ideally in a Gist: gist.github.com)
* pom.xml file or similar (also in a Gist)

#### Contributing

If you'd like to help us fix the issue by contributing some code, but would
like guidance or help in doing so, please mention it!

**Comment by agibsonccc** [comment]
Need to add this it appears: https://www.tensorflow.org/api_docs/python/tf/keras/losses/log_cosh
